Engine/Component/ComponentMaterial.cpp

```diff
@@ -49,6 +49,7 @@ void ComponentMaterial::Save(Config& config) const
 	config.AddFloat(k_specular, "kSpecular");
```

Comment:
I think mario removed the componentMaterial and we have to use the new one, we have to move everything from component Material to the new one